scripts/keyword-volume/design-pdf.css

```
@@ -0,0 +1,14 @@
+@page { size: Letter; margin: 20mm 18mm; }
```

Stylelint (17.14.0)

[error] Expected "Letter" to be "letter" (value-keyword-case)

(value-keyword-case)